ui/page/components/wallet_setup_page.go

```diff
@@ -120,6 +122,17 @@ func NewCreateWallet(l *load.Load, walletCreationSuccessCallback func(), assetTy
 
 	pg.materialLoader = material.Loader(l.Theme.Base)
 
+	defaultIndex := &cryptomaterial.DropDownItem{
```

Comment:
`defaultIndex` sounds a bit confusing, consider renaming to `defaultWordSeedType`